lib/array.js

```diff
@@ -11,11 +11,10 @@ const splitAt = (
   return [part1, part2];
 };
 
-const shuffle = (
-  // Shuffle an array
-  arr // array
-  // Returns: array
-) => (
+// Shuffle an array
+//   arr - array
+// Returns: array
+const shuffle = arr => (
```

Comment:
Better remove `()` in such cases
```suggestion
const shuffle = arr => arr.sort(() => Math.random() - 0.5);
```